get-started/part4.md

```diff
@@ -443,6 +445,20 @@ docker stack rm getstartedlab
 > manager, but _you'll need this swarm for part 5, so please keep it
 > around for now_.
 
+### Unsetting docker-machine shell variable settings
```

Comment:
Looks good but it might be easier to find this command if you place it after this section with a short note that this is how you unset: 

> Run the given command to configure your shell to talk to myvm1.
> 
> `eval $(docker-machine env myvm1)`